editor/gui/editor_toaster.cpp

```diff
@@ -371,6 +371,20 @@ Control *EditorToaster::popup(Control *p_control, Severity p_severity, double p_
 	p_control->set_h_size_flags(SIZE_EXPAND_FILL);
 	hbox_container->add_child(p_control);
 
+	// Copy button.
+	if (p_time > 0.0) {
+		Button *copy_button = memnew(Button);
+		copy_button->set_flat(true);
+		copy_button->set_button_icon(get_editor_theme_icon(SNAME("ActionCopy")));
+		copy_button->set_focus_mode(FOCUS_NONE);
+		copy_button->set_disabled(true);
+		copy_button->connect(SceneStringName(mouse_entered), callable_mp(this, &EditorToaster::_toggle_copy_button).bind(copy_button, false));
+		copy_button->connect(SceneStringName(mouse_exited), callable_mp(this, &EditorToaster::_toggle_copy_button).bind(copy_button, true));
```

Comment:
What's this for?

Reply:
So I have the button disabled by default, and it re-enables when hovered (_or_ we can change it to when the panel is hovered). I thought this looked a little less noisy then having both buttons enabled, but I'm down to change to whatever.

https://github.com/user-attachments/assets/3486003a-e3cd-475d-b2c8-0a0481147722

**VS.**

https://github.com/user-attachments/assets/0012c637-73a4-483a-8221-5606d48dc196

Reply:
I was wondering why the button is gray. I think they should both be enabled.

Reply:
Fair, I'll change it rn.